components/kserve/kserve_config_handler.go

```diff
@@ -0,0 +1,163 @@
+package kserve
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+
+	"github.com/hashicorp/go-multierror"
+	operatorv1 "github.com/openshift/api/operator/v1"
+	corev1 "k8s.io/api/core/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	dsciv1 "github.com/opendatahub-io/opendatahub-operator/v2/apis/dscinitialization/v1"
+	"github.com/opendatahub-io/opendatahub-operator/v2/pkg/deploy"
+	"github.com/opendatahub-io/opendatahub-operator/v2/pkg/feature"
+)
+
+const (
+	KserveConfigMapName string = "inferenceservice-config"
+)
+
+func (k *Kserve) setupKserveConfigAndDependencies(ctx context.Context, cli client.Client, dscispec *dsciv1.DSCInitializationSpec) error {
+	// as long as Kserve.Serving is not 'Removed', we will setup the dependencies
+
+	switch k.Serving.ManagementState {
+	case operatorv1.Managed:
+		// check on dependent operators if all installed in cluster
+		dependOpsErrors := checkDependentOperators(cli).ErrorOrNil()
+		if dependOpsErrors != nil {
+			return dependOpsErrors
+		}
```

Comment:
This looks also related to my other comment about the rebase mistake.

It may have been the case that the call to `checkDependentOperators` was moved around in a previously merged PR, and may no longer relevant here (because `configureServerless` would already do the call when needed).

Reply:
Fixed